internal/collector.go

```diff
@@ -0,0 +1,55 @@
+package internal
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const Name = "kafka_cronsumer_"
+
+type Collector struct {
+	kafkaCronsumer kafkaCronsumer
+
+	totalRetriedMessagesCounter *prometheus.Desc
+	totalIgnoredMessagesCounter *prometheus.Desc
+}
+
+func (s *Collector) Describe(ch chan<- *prometheus.Desc) {
+	prometheus.DescribeByCollect(s, ch)
+}
+
+func (s *Collector) Collect(ch chan<- prometheus.Metric) {
+	producerMetric := s.kafkaCronsumer.GetMetric()
+
+	ch <- prometheus.MustNewConstMetric(
```

Comment:
We should also add these exposed metrics in README, like [that](https://github.com/Trendyol/kafka-konsumer#exposed-metrics)